pkg/util/lock/metric_mutex.go

```diff
@@ -0,0 +1,102 @@
+package lock
+
+import (
+	"sync"
+	"time"
+
+	"github.com/cockroachdb/errors"
+	"github.com/milvus-io/milvus/internal/querycoordv2/params"
```

Comment:
Don't use the params of this package, just use paramtable.Get().

Reply:
changed